src/Commands/Tag/ListCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Tag;
+
+use Consolidation\OutputFormatters\StructuredData\AssociativeList;
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+class ListCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Lists the tags which an organization has added to a site
+     *
+     * @authorized
+     *
+     * @command tag:list
+     * @aliases tags
+     *
+     * @param string $site_name The name or UUID of a site to list the tags of
+     * @param string $organization The name or UUID of an organization which has tagged this site
+     *
+     * @return AssociativeList
```

Comment:
I left it out because this model only has the tag name itself, so I thought that a bulleted list (which comes out in other formats as a simple array) would work nicely.